hadoop-hdds/docs/content/_index.md

```diff
@@ -34,11 +34,12 @@ ref "JavaApi.md"
 >}}), [S3 protocol support] ({{< ref "S3.md" >}}), and a [command line interface]
 ({{< ref "shell/_index.md" >}})  which makes it easy to use Ozone.
 
-Ozone consists of volumes, buckets, and Keys:
+Ozone consists of volumes, buckets and keys:
```

Comment:
Well, use of a comma before and is an accepted usage. 
https://www.businessinsider.com/a-guide-to-proper-comma-use-2013-9

Reply:
:) I guess I lean too heavily on British English. Though this one is trivial. I can undo this one.